packages/backend/src/open_payments/quote/service.ts

```diff
@@ -288,77 +292,58 @@ export async function startQuote(
   }
 }
 
-type QuoteHeaders = {
-  Accept: string
-  'Content-Type': string
-  'Rafiki-Signature'?: string
+function getFees(fee: Fee | undefined, principal: bigint): bigint {
```

Comment:
Maybe this method can live in feeService? just a suggestion

Reply:
I added the method to the model since we already had the fee model here. It felt wrong to pass the fee into the fee service like `feeService.calculate(fee, principal)` or to re-lookup the fee. Later if we just have the `feeId` or `assetId` we can do `feeService.calculate(id, principal)` which gets the fee then calls the `calculate` method.